packages/cli/src/cmds/account/cmds/validator/options.ts

```diff
@@ -18,4 +19,11 @@ export const accountValidatorOptions: ICliCommandOptions<IAccountValidatorArgs>
     defaultDescription: defaultAccountPaths.secretsDir,
     type: "string",
   },
+
+  publicKeysFile: {
+    description: "File for storing public keys",
```

Comment:
- What is the format and structure of this file?
- Is the user expected to provide this file or it generated automatically?
- Do Lighthouse and Teku use a similar file for their remote signer?

Reply:
Teku has 2 ways to input the public keys either through the `cli` or using a `yaml` file. Our initial idea was to store the public keys in a text file which will be created by the user & retrieve it from there.
But I think what Teku does is clean we can pursue with their implementation. What do you think ? 
@dapplion

Reply:
Following Teku's approach is fine for me :+1: Be aware of a parallel imitative to populate that file via an API too https://github.com/ethereum/keymanager-APIs/pull/17